docs/mining/overview.md

```diff
@@ -26,74 +26,141 @@ transactions waiting to be processed in the mempool.
+:fontawesome-solid-exclamation-triangle: **Solo mining is not recommended and is not covered by this documentation!** The Decred network regularly sees a network hash rate of up to 422Ph/s. Solo mining is generally only done by advanced individuals or organized groups with a large cluster of GPUs so it is not addressed here.
 
-Please be certain to select the correct operating system (Windows/Linux) and the correct version for your GPU type (CUDA for NVIDIA cards, OpenCL/OpenCLADL for AMD cards). gominer is only available to 64-bit operating systems. User guides to start mining with gominer can be found here:
 
-- [gominer Pool-Mining](../mining/proof-of-work/pool-mining/gominer.md)
+#### Pool Mining
 
-### Unofficial Miners
+When you mine in a pool, your hashrate is combined with all the other pool miners’ hashrates to look for the correct solution for a block. You will receive a reward based on the amount of work your miner performs in the pool. Most mining pools pay on a pay-per-share method meaning that miners earn a steady amount of Decred rather than the “all or none” of solo mining.
 
-* :octicons-mark-github-16: [cgminer](https://github.com/kR105-zz/cgminer) - cgminer is a popular miner for **AMD** GPUs that has a long history of use in many different cryptocurrencies. It is more difficult to use than the decred gominer.
 
-* :octicons-mark-github-16: [ccminer](https://github.com/tpruvot/ccminer) - ccminer is a popular miner for **NVIDIA** GPUs that has a long history of use in many different cryptocurrencies. It is more difficult to use than the decred gominer.
+## Choosing and configuring a mining pool 
 
-* :octicons-mark-github-16: [sgminer](https://github.com/tpruvot/sgminer) - Those with an **AMD** graphics card running Windows may want to download sgminer.
+To finish the configuration, you will need to choose a mining pool. You can consider payout method ([PPS vs PPLNS](https://www.hashrateindex.com/blog/pps-fpps-pplns-pps_plus)), pool fee, geographic location of the pool, ping time to the server, user interface, statistics / data, customer support, community channels and more. Mining pools all work more or less the same but you may wish to sign up at multiple pools and see which one suits you the best.
 
----
+These mining pools are known to support Decred:
+
+
+
+*   **[Luxor](https://mining.luxor.tech/coins/decred)** (USA)
+*   **[F2Pool](https://www.f2pool.com/)** (China)
+*   **[Poolin](https://www.poolin.com/)** (China)
+*   **[UUpool](https://uupool.cn/dcr)** (China)
+*   **[Antpool](https://antpool.com/)** (China)
+*   **[BTC.com](https://btc.com/)** (China)
```

Comment:
Let's use the URL https://pool.btc.com/ for this one

Reply:
Updated URL for BTC.com